test/test_all_options.jl

```diff
@@ -49,8 +57,13 @@ dt = srcGeometry.dt[1]
                 y_hat = J*dm
                 x_hat2 = adjoint(J)*y0
 
-                c = dot(y0, y_hat)
-                d = dot(dm, x_hat2)
+                if ~parsed_args["viscoacoustic"]
+                    c = dot(y0, y_hat)
+                    d = dot(dm, x_hat2)
+                else
+                    c = dot(y0.data[1], y_hat.data[1])
```

Comment:
shoudln't be needed

Reply:
We will create an MFE to detail this problem.

Reply:
@mloubout  please see #104